src/components/social/PostCard.tsx

```
@@ -383,68 +370,33 @@ export function PostCard({
           <div className="relative">
-            <Dialog
+            <div
+              className="relative cursor-pointer group max-h-[400px] w-full"
+              onClick={() => setIsImageDialogOpen(true)}
+            >
+              <Image
+                src={displayPost.image_urls[currentImageIndex]}
+                alt="投稿画像"
+                fill
+                className="object-cover rounded-lg"
+                sizes="(max-width: 768px) 100vw, 50vw"
```

Biome (2.3.13)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)